_includes/_guides/_solo_crunching.htm

```diff
@@ -0,0 +1,277 @@
+                  </li>
+              </ol>
+
+              <p>
+                  If you don't want to buy any Gridcoin, you can get some through
+                  the faucets or through <a href="#">pool mining</a>.
+                  On most exchanges, you need to buy Bitcoin first in order to
+                  buy Gridcoin.
+              </p>
+          </div><!-- .step-body -->
+      </section><!-- .step -->
+
+      <section class="step">
+          <header>
+              <img src="/assets/img/ic-guide-earn-grc.svg" alt="">
+              <h3>
+                  <span class="echo" aria-hidden="true">07</span>
+                  <strong>Step 7.</strong> Earn Research Rewards
+              </h3>
+          </header>
+          <div class="step-body">
+              <h4 class="h5">Check Your Projects</h4>
+              <ol>
+                  <li>
+                      Make sure any projects you crunch are on the
+                      <a href="/whitelist.htm">whitelist</a>. If a project
+                      isn't approved, you will not be rewarded for it.
+                  </li>
+                  <li>
+                      Check that each project crunched is on team Gridcoin.
+                      You will not earn rewards if they are not on the team.
+                  </li>
+                  <li>
+                      Check that the CPIDs (cross-project identifiers) are the
+                      same across your projects. You can check on each project
+                      or check using
+                      <a href="https://www.boincstats.com/">BOINCstats</a>.
+                      If you found differing CPIDs, use this
+                      <a href="https://steemit.com/gridcoin/@dutch/the-definitive-guide-to-fixing-a-split-cpid-gridcoin-s-and-boinc-s-1-problem">guide</a>.
+                  </li>
+                  <li>
+                      If you installed the BOINC client anywhere other than the
+                      default installation location, you'll need to specify
+                      where you installed BOINC in the gridcoinresearch.conf file.
+                      <br />
+                      <br />
+                      <b>Windows</b>: (Note the double backslashes are necessary. Change location according to your non-default install location)
+                      <pre class="highlight">boincdatadir=D:\\ProgramData\\BOINC\\
+boincappdir=C:\\Program Files\\BOINC\\</pre>
+                      <br />
+                      <b>Linux:</b>
+                      <pre class="highlight">boincdatadir=/var/lib/boinc-client/</pre>
+                  </li>
+              </ol>
+
+              <h4 class="h5">Configure the Wallet</h4>
+              <ol>
+                  <li>
+                      In the wallet, go to the "overview" page and check the
+                      message in the lower-left. Make sure it doesn't say
+                      "Staking Only - Investor Mode".
```

Comment:
The new GUI has a badge that's always there in the top-left to indicate which mode you are in. This is probably a better way to check it
```suggestion
                      In the wallet, go to the "overview" page and check the
                      badge in the top-left. Make sure it doesn't say
                      "Staking Only".
```